Cargo.toml

```diff
@@ -14,13 +14,13 @@ targets = ["x86_64-unknown-linux-gnu"]
 [dependencies]
 anyhow = { version = "1.0", default-features = false }
 bcs = { git = "https://github.com/eigerco/bcs.git", default-features = false, branch = "master" }
-codec = { package = "parity-scale-codec", version = "3.6.1", default-features = false, features = ["derive",] }
-scale-info = { version = "2.5.0", default-features = false, features = ["derive"] }
-frame-benchmarking = { version = "4.0.0-dev", default-features = false, optional = true, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
-frame-support = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
-frame-system = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
 serde = { version = "1.0", default-features = false, features = ["derive"] }
-sp-std = { default-features = false, git = 'https://github.com/paritytech/substrate.git', branch = 'polkadot-v1.0.0' }
+codec = { package = "parity-scale-codec", version = "3.6", default-features = false, features = ["derive"] }
+scale-info = { version = "2.5", default-features = false, features = ["derive"] }
```

Comment:
Upstream updated `scale-info` codec to 2.10.0:
https://github.com/paritytech/polkadot-sdk/blob/fcfdb98ab5c5fce91cce339c8d3e4f5f5844bbe1/substrate/bin/node-template/pallets/template/Cargo.toml#L19

Reply:
Done, good catch!